game/scripts/npc/abilities/lina_laguna_blade.txt

```diff
@@ -43,7 +43,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "450 650 850 1850 3000"
+        "damage"                                          "450 750 1050 1850 2650" //OAA
```

Comment:
1050 pure dmg on lvl 3? hmm. Also don't nerf lvl 5.

Reply:
I think the level 2 and 3 ultimates should stay the same. Everyone just has more HP in OAA and this would make the hero much stronger in the mid game. Also The level 5 ultimate needs reducing. The main reason why lina isn't op is because you very rarely get to level 50. And at level 5 laguna blade will just oneshot heroes without 10K hp